roles/users/defaults/main.yml

```diff
@@ -1,5 +1,3 @@
 deploy_user: deploy
 deploy_group: deploy
 deploy_groups: www-data
```

Comment:
Yeah I probably can. That was left over since I needed those vars to be globally available.